src/styles/components/EventDescriptionModal.module.css

```diff
@@ -1,27 +1,43 @@
 .container {
-  width: 100svw;
-  height: 100svh;
-  position: fixed;
-  inset: 0;
-  z-index: 999;
-  background-color: rgba(10, 10, 10, 0.6);
+  display: flex;
+  gap: 1rem;
+  align-items: stretch;
+  justify-content: center;
 }
 
 .outerModalContainer {
-  display: grid;
+  display: flex;
   grid-template-columns: 50% 0 50%;
   grid-template-rows: 1fr;
   max-height: 90svh;
+  padding: 1rem;
 }
 
 .window {
-  position: absolute;
+  z-index: 99;
+  /* position: absolute;
   top: 50%;
   left: 50%;
```

Comment:
yeah i see all of this is commented out, was there a resason for hat?

Reply:
I believe they either did not affect the output visually, but I can confirm this later when events are displayed on the website again.